build/yamls/collector/templates/ipfix-collector.yaml

```diff
@@ -0,0 +1,57 @@
+apiVersion: v1
```

Comment:
Lets use "template" or "base" instead of "templates". There is only one template.

Reply:
I also feel uncomfortable about that. But unfortunately the folder has to be named as "templates" to make it configurable. It has to follow the Helm chart structure as mentioned in the similar issue: https://github.com/helm/helm/issues/4152
I have tried to rename it and the configuration doesn't work as a result. I'm thinking maybe we can still switch back to kustomize in the future if there's a lot of inconvenience.